official/resnet/cifar10_main.py

```diff
@@ -23,6 +23,7 @@
 from absl import app as absl_app
 from absl import flags
 import tensorflow as tf  # pylint: disable=g-bad-import-order
+from tensorflow.python.platform import gfile
```

Comment:
Drive by comment: this is exported with tf. (so you can do `tf.gfile.Exists(...)`)

Reply:
ahhh sweet I will swap that out.

Reply:
HIGH FIVE drive-by

Reply:
fixed